### PR TITLE
Add manage integration section for k8s-api-operator

### PR DESCRIPTION
**Pull request by sajithaliyanage**

### CODE REVIEW

File: en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
+# Manage Integrations
+
+The Kubernetes API operator (**k8s-api-operator**) provides first-class support for Micro Integrator deployments in the Kubernetes ecosystem. It uses the **Integration custom resource** (`integration_cr.yaml` file) that is available in the Kubernetes exporter module (exported from WSO2 Integration Studio) and deploys the integration in your Kubernetes environment.
+
+The operator is configured with an NGINX Ingress controller by default, which exposes the deployed integration through HTTP. If required, you can use the operator's configuration mapper (`integration_controller_conf.yaml` file) to disable the default Ingress controller and apply other configuration changes.
```
Comment (abeykoon) [inline en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md:5]: HTTP/HTTPs protocols.

Comment (abeykoon) [inline en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md:5]: to update ingress controller configurations. Also, you can use the same file to disable ingress controller creation when applying the operator if you plan to use some other existing ingress controller.

File: en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+!!! Tip
+    To try the end-to-end process of deploying integration solutions on Kubernetes, see the integration examples: 
+
+    - [Hello World example](../integration-samples/hello-world)
+    - [Message Routing example](../integration-samples/content-based-routing)
+    - [JMS Sender/Receiver exampe](../integration-samples/jms-sender-receiver)
+
+Given below are the main steps your will follow when you deploy integration solutions in a Kubernetes cluster.
+
+1.  Be sure that the [system requirements](#prerequisites-system-requirements) are fulfilled, and that the [EI K8s operator](#install-the-ei-k8s-operator) is installed in your Kuberenetes environment.
```
Comment (abeykoon) [inline en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md:29]: Now APIM operator.

File: en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+
+    ```bash
+    curl http://<HOST-NAME>/<INTEGRATION-NAME>-inbound/<PORT>/<CONTEXT>
+    ```
+
+-   HTTPS request
+
+    ```bash
+    curl --cacert <CERT_FILE> https://<HOST-NAME>/<INTEGRATION-NAME>-inbound/<PORT>/<CONTEXT>
+    ```
```
Comment (abeykoon) [inline en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md:340]: It worth mentioning below. 

- all ingress controller related configs are controlled by `integration-ingress-config` configMap defined at `integration_controller_conf.yaml` file. 
- If `minReplicas` and `maxReplicas` are not defined in `` the operator will get pick up what is defined at `integration_controller_conf.yaml` file as defaults. 
- If `autoScale. enabled` is not defined at `` the operator will get pick up what is defined at `integration_controller_conf.yaml` file as the default. 
- All Horizontal Pod scaling configurations are defined under `hpaMetrics` at `integration_controller_conf.yaml` file.

Finally, 
- You can update the `reconcileInterval` (how frequent operator should scan for artefact updates) at `integration_controller_conf.yaml` file.

File: en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md (hunk 1 above)
Comment (sajithaliyanage) [reply en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md]: Fixed

Comment (sajithaliyanage) [reply en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md]: Fixed

File: en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md (hunk 2 above)
Comment (sajithaliyanage) [reply en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md]: Fixed

File: en/mkdocs.yml [hunk 4]
```diff
@@ -1028,6 +1023,11 @@ nav:
         - Kubernetes Operators:
           - K8s API Operator:
               - K8s API Operator Overview: install-and-setup/setup/kubernetes-operators/k8s-api-operator/k8s-api-operator.md
```
Comment (nilminiwso2) [inline en/mkdocs.yml:1025]: This structure will break the build. When there are child pages, the parent title cannot have a page of its own. Let's have it like this?

- K8s Operator:
  - Operator Overview: <path>
  - Deploy Integrations using K8s Operator: <path>
  - Integration Deployment Samples:
    - Sample 1.. : <path>
    - Sample 2...: <path>
    - Sample 3...: <path>

File: en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md [hunk 5]
```diff
@@ -0,0 +1,453 @@
...
+##  Prerequisites (system requirements)
+
+Listed below are the system requirements for deploying integration solutions in Kubernetes using the K8s API Operator.
+
+!!! Info
+    The K8s API Operator (k8s-api-operator) is built with **operator-sdk v0.16.0** and supported in the below environments.
+
+- [Kubernetes](https://kubernetes.io/docs/setup/) cluster and **v1.14+** client. 
+- [Docker](https://docs.docker.com/)
+- Install K8s API Operator using OperatorHub.io. For information, see the installation instructions in the [OperatorHub.io](https://operatorhub.io/operator/api-operator)
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md:18]:
```suggestion
- [K8s API Operator][OperatorHub.io](https://operatorhub.io/operator/api-operator)
```

File: en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md [hunk 6]
```diff
@@ -0,0 +1,453 @@
...
+
+When you run the above command, Kubernetes will spawn another temporary pod, which has the same behaviour of the pod we have deleted. Then the deleted pod will restart by pulling the latest tag from the Docker image path. 
+
+!!! Note 
+    Here we are recommending to use a different image path for the updated integration solution. Otherwise, (because the Docker image is re-pulled from the existing deployment) some traffic from outside might get lost. 
+
+
+## Run inbound endpoints
+
+[Inbound Endpoints](../../../references/synapse-properties/inbound-endpoints/about-inbound-endpoints/) in the Micro Integrator are used for separating endpoint listeners. That is, for each HTTP inbound endpoint, messages are handled separately. Also, we can create any number of inbound endpoints on any port.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md:420]:
```suggestion
[Inbound Endpoints]({{base_path}}/reference/synapse-properties/inbound-endpoints/about-inbound-endpoints/) in the Micro Integrator are used for separating endpoint listeners. That is, for each HTTP inbound endpoint, messages are handled separately. Also, we can create any number of inbound endpoints on any port. 
```

**Review by nilminiwso2 — APPROVED** [review]

**Review by abeykoon — APPROVED** [review]

**Pull request merged**